android/app/src/main/java/app/organicmaps/util/StringUtils.java

```diff
@@ -24,6 +25,22 @@ public static String formatUsingSystemLocale(String pattern, Object... args)
     return String.format(Locale.getDefault(), pattern, args);
   }
 
+  /**
+   * This method returns correct string representation of % (percent number) for different locales
+   * For example, that's how the result of formatPercent(23.19) would look like:
+   * — 23.19% (in US locale)
+   * — %23.19 (in Turkish locale)
+   * — 23,19% (in Latvian locale)
+   *
+   * @param percent
+   * @return correct string representation of percent for different locales
+   */
+  public static String formatPercent(double percent) {
+    NumberFormat percentFormat = NumberFormat.getPercentInstance();
+    percentFormat.setMaximumFractionDigits(2);
+    return percentFormat.format(percent);
+  }
```

Comment:
May I ask someone who has experience in iOS development to do similar thing there?

```
let value = 0.7523
let percentFormatter = NumberFormatter()
percentFormatter.numberStyle = .percent
percentFormatter.maximumFractionDigits = 2
let formatted = percentFormatter.string(from: NSNumber(value: value)) // Outputs: "%75,23" in Turkish locale
```

@pastk @vng

Reply:
@kirylkaveryn